Build break in master

```none
R:\roslyn\src\VisualStudio\Core\Def\Implementation\LanguageClient\InProcLanguageServer.cs(155,16): error CS0103: The na
me '_protocol' does not exist in the current context [R:\roslyn\src\VisualStudio\Core\Def\Microsoft.VisualStudio.Langua
geServices_z3ntnm0v_wpftmp.csproj]
R:\roslyn\src\VisualStudio\Core\Def\Implementation\LanguageClient\InProcLanguageServer.cs(160,16): error CS0103: The na
me '_protocol' does not exist in the current context [R:\roslyn\src\VisualStudio\Core\Def\Microsoft.VisualStudio.Langua
geServices_z3ntnm0v_wpftmp.csproj]
R:\roslyn\src\VisualStudio\Core\Def\Implementation\LanguageClient\InProcLanguageServer.cs(216,16): error CS0103: The na
me '_protocol' does not exist in the current context [R:\roslyn\src\VisualStudio\Core\Def\Microsoft.VisualStudio.Langua
geServices_z3ntnm0v_wpftmp.csproj]
    0 Warning(s)
    3 Error(s)
```

--- a/src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs
+++ b/src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs
@@ -152,12 +152,12 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
 
         [JsonRpcMethod(Methods.TextDocumentCodeActionName, UseSingleObjectParameterDeserialization = true)]
         public Task<VSCodeAction[]> GetTextDocumentCodeActionsAsync(CodeActionParams codeActionParams, CancellationToken cancellationToken)
-            => _protocol.ExecuteRequestAsync<CodeActionParams, VSCodeAction[]>(Methods.TextDocumentCodeActionName,
+            => _requestHandlerProvider.ExecuteRequestAsync<CodeActionParams, VSCodeAction[]>(Methods.TextDocumentCodeActionName,
                 codeActionParams, _clientCapabilities, _clientName, cancellationToken);
 
         [JsonRpcMethod(MSLSPMethods.TextDocumentCodeActionResolveName, UseSingleObjectParameterDeserialization = true)]
         public Task<VSCodeAction> ResolveCodeActionAsync(VSCodeAction vsCodeAction, CancellationToken cancellationToken)
-            => _protocol.ExecuteRequestAsync<VSCodeAction, VSCodeAction>(MSLSPMethods.TextDocumentCodeActionResolveName,
+            => _requestHandlerProvider.ExecuteRequestAsync<VSCodeAction, VSCodeAction>(MSLSPMethods.TextDocumentCodeActionResolveName,
                 vsCodeAction, _clientCapabilities, _clientName, cancellationToken);
 
         [JsonRpcMethod(Methods.TextDocumentCompletionName, UseSingleObjectParameterDeserialization = true)]
@@ -213,7 +213,7 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
 
         [JsonRpcMethod(Methods.WorkspaceExecuteCommandName, UseSingleObjectParameterDeserialization = true)]
         public Task<object> ExecuteWorkspaceCommandAsync(ExecuteCommandParams executeCommandParams, CancellationToken cancellationToken)
-            => _protocol.ExecuteRequestAsync<ExecuteCommandParams, object>(Methods.WorkspaceExecuteCommandName,
+            => _requestHandlerProvider.ExecuteRequestAsync<ExecuteCommandParams, object>(Methods.WorkspaceExecuteCommandName,
                 executeCommandParams, _clientCapabilities, _clientName, cancellationToken);
 
         [JsonRpcMethod(Methods.WorkspaceSymbolName, UseSingleObjectParameterDeserialization = true)]

